### DISCUSSION TITLE
Add Constant Speed mod to osu!taiko

### DISCUSSION THREAD

**Discussion by Hiviexd** [ppy/osu#17927] [Ideas]
osu!mania currently has a Constant Speed mod which nullifies all SV changes and makes the scroll speed constant. 
I think it would also be a welcome addition to osu!taiko, because a difficulty with drastic SV changes, and the same difficulty with no SV changes can provide drastically different experiences and enhances accessibility for players.

**Comment by zachmanthethird** [comment]
[Interesting tangent discussion in the mapping forums](https://osu.ppy.sh/community/forums/topics/1562418) that might be worth discussing here as well. There may be necessary SV included in the map (to indicate BPM changes, for example) that may make the lack of such SV more difficult. However, the validity and necessity of that will only be determined once the new taiko SR system is set in place.
